CAP/gap/CAP.gd

```diff
@@ -378,19 +378,22 @@ DeclareGlobalFunction( "EnableBasicOperationTypeCheck" );
 
 #############################################
 ##
-#! @Section Disable add in functions
+#! @Section Enable add in functions
 ##
 #############################################
 
 #! @BeginGroup
 #! @Description
-#!  Enables/disables the call of add for the output
+#!  Enables/disables the call of <C>Add</C> for the output
 #!  of primitively added functions for the category <A>C</A>.
-#!  This can be savely done if the all objects and morphisms
-#!  are added to the category by their constructors.
-#! @Arguments C
-DeclareGlobalFunction( "DisableAddForCategoricalOperations" );
+#!  If this is disabled (default), the output of primitively added functions must
```

Comment:
Can you not use `If this is enabled`, but something like `Enables ...`/`Disables ...`?

It just seems a bit weird to read it that way, as the docstring should document what the function does, and not what the effects are. It is not completely unclear here, but nevertheless.

Reply:
I didn't see the first line, sorry.